harbors/management/commands/add_helsinki_harbors_images.py

```diff
@@ -0,0 +1,42 @@
+import os
+
+from django.conf import settings
+from django.core.files import File
+from django.core.management.base import BaseCommand
+
+
+from harbors.models import Harbor
+
+
+class Command(BaseCommand):
+
+    def handle(self, **options):
+        updated_harbors = 0
+        missing_images = []
+
+        for harbor in Harbor.objects.filter(servicemap_id__isnull=False):
```

Comment:
I still have a slight problem with this, as `servicemap_id` can be both `null` and `""`, I think more robust would be to check both of those.

Reply:
ok, had the same thought...

Reply:
fixed :)